kuksa_databroker/databroker/src/broker.rs

```diff
@@ -921,6 +961,117 @@ impl QuerySubscription {
     }
 }
 
+impl ContinuousSubscription {
+    async fn notify(
```

Comment:
I see that this is very similar to the ChangeSubscription notify. Couldn't it be reused somehow? What I mean is just use different subscribe function but maybe one notify? If this is not doable let me know :)